x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/details_step/details_step_form.tsx

```diff
@@ -188,15 +188,30 @@ export const DetailsStepForm: FC<CreateAnalyticsStepProps> = ({
         />
       </EuiFormRow>
       <EuiFormRow
-        isInvalid={createIndexPattern && destinationIndexPatternTitleExists}
-        error={
-          createIndexPattern &&
-          destinationIndexPatternTitleExists && [
-            i18n.translate('xpack.ml.dataframe.analytics.create.indexPatternExistsError', {
-              defaultMessage: 'An index pattern with this title already exists.',
-            }),
-          ]
+        fullWidth
+        isInvalid={
+          (createIndexPattern && destinationIndexPatternTitleExists) || !createIndexPattern
         }
+        error={[
+          ...(createIndexPattern && destinationIndexPatternTitleExists
+            ? [
+                i18n.translate('xpack.ml.dataframe.analytics.create.indexPatternExistsError', {
+                  defaultMessage: 'An index pattern with this title already exists.',
+                }),
+              ]
+            : []),
+          ...(!createIndexPattern
+            ? [
+                i18n.translate(
+                  'xpack.ml.dataframe.analytics.create.shouldCreateIndexPatternMessage',
+                  {
+                    defaultMessage:
+                      'You may not be able to view job results if an index pattern is not created for the destination index.',
```

Comment:
Is there an easy way to switch the color of this message to the EUI `warning` text color, seeing it is only a warning and not an 'error'?

Reply:
Good point! Updated that message to have warning color in b7c65a8d71b075768b2c7d4434c260087de30c8a